src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.SearchResult.cs

```diff
@@ -66,22 +62,20 @@ private class SearchResult : INavigateToSearchResult
 
             private static readonly char[] s_dotArray = { '.' };
 
-            private static string ConstructSecondarySortString(
-                Document document,
-                DeclaredSymbolInfo declaredSymbolInfo)
+            private string ConstructSecondarySortString()
             {
                 var parts = ArrayBuilder<string>.GetInstance();
```

Comment:
:question: Why not a `StringBuilder`? It would even be pretty easy to know the required length in advance.